build.sbt

```diff
@@ -26,17 +26,18 @@ ThisBuild / crossScalaVersions := Seq(
 
 lazy val root = tlCrossRootProject.aggregate(lib)
 
-lazy val munitVersion      = "1.0.0"
-lazy val scalacheckVersion = "1.18.0"
+lazy val munitVersion           = "1.0.1"
+lazy val scalacheckVersion      = "1.18.0"
+lazy val munitScalacheckVersion = "1.0.0"
 
 lazy val lib = crossProject(JVMPlatform, JSPlatform)
   .crossType(CrossType.Pure)
   .in(file("lib"))
   .settings(
     name := "scalac-options",
     libraryDependencies ++= Seq(
-      "org.scalameta"  %%% "munit"            % munitVersion      % Test,
-      "org.scalacheck" %%% "scalacheck"       % scalacheckVersion % Test,
-      "org.scalameta"  %%% "munit-scalacheck" % munitVersion      % Test
+      "org.scalameta"  %%% "munit"            % munitVersion           % Test,
+      "org.scalacheck" %%% "scalacheck"       % scalacheckVersion      % Test,
+      "org.scalameta"  %%% "munit-scalacheck" % munitScalacheckVersion % Test
```

Comment:
Looks like Scalameta diverged *munit* and *munit-scalacheck* versions eventually. Disuniting them here as well.